plugins/link/SharedMemory.h

```diff
@@ -17,22 +17,29 @@
 #	include <string>
 #endif
 
+struct LinkedMem;
+
 class SharedMemory {
 public:
 	explicit SharedMemory();
 	~SharedMemory();
 
-	std::size_t size();
-
 	void close();
 
 	int lastError() const;
 
-	void *mapMemory(const char *name, std::size_t size);
+	bool mapMemory(const char *name);
+
+	bool isMemoryMapped();
+
+	void read(LinkedMem &dest);
+
+	void write(const LinkedMem &source);
+
+	void fillWithZero();
```

Comment:
I think conceptually, this should be a private function :thinking: 
It is only used inside member functions, isn't it?

Reply:
It's also used in `mumble_shutdownPositionalData`.